### PR TITLE
feat: add specsFile to clabverter

### PR DESCRIPTION
**Pull request by SimPeccaud**
`--specsFile` clabverter option allows to point to a yaml file containing values to be included in the output. More specifically in the `specs` field of the topology manifest.

The value of `specsFile` is to be able to share values of a shared cluster across all users, especially when not all users are comfortable with k8s/c9s.

### CODE REVIEW

**Comment by carlmontanari** [comment]
boom! very nice! will check it out and comment hopefully tomorrow but if not this weekend!

**Comment by SimPeccaud** [comment]
> boom! very nice! will check it out and comment hopefully tomorrow but if not this weekend!

thanks, in the meantime I'll have a look at that lint error

File: clabverter/test-fixtures/clabversiontest/specs.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+statusProbes:
+  enabled: true
+  excludedNodes:
+  - baguette
```
Comment (carlmontanari) [inline clabverter/test-fixtures/clabversiontest/specs.yaml:4]: 🥖 😁

File: cmd/clabverter/cli/entrypoint.go [hunk 2]
```diff
@@ -9,6 +9,7 @@ import (
 
 const (
 	topologyFile         = "topologyFile"
+	specsFile            = "specsFile"
```
Comment (hellt) [inline cmd/clabverter/cli/entrypoint.go:12]: I am rooting for shorter arg names, but for this feature I fail to find one.
Maybe though it should be `--topoSpecFile` to be precise what it is about.

**Pull request merged**